scripts/dev/check_parameters.php

```diff
@@ -372,6 +373,9 @@ function recurse($path)
     }
 }
 
+$GLOBALS['error_reported'] = false;
```

Comment:
Why not instead:
```suggestion
$error_reported = false;
```
Analogous below.